okhttp/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
   compileOnly deps.openjsse
   compileOnly deps.jsr305
   compileOnly deps.animalSniffer
+  compileOnly deps.guava
```

Comment:
We are not adding a Guava dependency.

Reply:
I assumed it would be fine since it's already part of the build.gradle here: https://github.com/square/okhttp/blob/master/build.gradle#L13

What's the concern with adding it? And do you have any suggestions otherwise?